packages/pg-v5/commands/upgrade.js

```diff
@@ -9,25 +9,25 @@ async function run(context, heroku) {
   let {app, args, flags} = context
   let db = await fetcher.addon(app, args.database)
 
-  if (util.essentialPlan(db)) throw new Error('pg:upgrade is only available for follower databases on at least the Standard tier.')
+  if (util.legacyEssentialPlan(db)) throw new Error('pg:upgrade is only available for Essential-* databases and follower databases on Standard-tier and higher plans.')
 
-  let [replica, status] = await Promise.all([
+  let [replica] = await Promise.all([
```

Comment:
Is there a reason to continue to use `Promise.all` here? There is only one item in the array.

Reply:
No there's not. Good catch! Updated!